### DISCUSSION TITLE
404

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#16] [备发布站的评论系统]
# 404

🌱 oldwinterの数字花园

https://notes.oldwinter.top/%E5%B8%B8%E9%9D%92%E7%AC%94%E8%AE%B0-evergreen-note

**Comment by zen010101** [comment]
怎么删除了？

**Reply by oldwinter** [reply]
朋友，你是本库多久前的用户啦？没有删除，原来这则笔记名叫 常青笔记-evergreen-note，我后来把英文换成别名了，只保留了中文名，现在就叫常青笔记。 https://notes.oldwinter.top/%E5%B8%B8%E9%9D%92%E7%AC%94%E8%AE%B0
